Add date/time text to notifications view

Turakar opened the pull request:
Ziemlich trivial glaube ich ;)

Es wird unterschieden zwischen:
1. Am gleichen Tag: Zeige Zeit an
2. Im gleichen Jahr: Zeige Tag und Monat an
3. Älter: Zeige Tag, Monat und Jahr an

Keine Lokalisierung auf andere Locales, wie US oder so, aber das ist glaube ich auch nicht nötig.

Turakar committed:
Add date/time text to notifications view

rleh commented:
Finde ich sehr schön :+1: 

Ich denke es wäre sinnvoll die Datum und Zeit Formatierung in eine Funktion in einer zu erstellenden Util Klasse auszulagern, damit Datum und Zeit in allen weiteren Ansichten in Zukunft in der sleben Formatierung angezeigt werden.

Ansonsten könnte von mir aus gemergt werden.

Turakar committed:
Organize date management better

- move MyTFG.getDate() to utils.TimeUtils (new) and rename to getDateStringComplete()
- move date/time code from StartFragment to utils.TimeUtils and name it getDateStringShort
- refactoring

Turakar committed:
Remove comment

rleh commented:
Kann meinetwegen gemergt werden :+1:
